api/transport/request_test.go

```diff
@@ -114,9 +115,11 @@ func TestValidator(t *testing.T) {
 			err = request.ValidateUnaryContext(ctx)
 		}
 
-		if tt.wantErr != "" {
+		if len(tt.wantMissingParams) > 0 {
 			if assert.Error(t, err) {
-				assert.Equal(t, tt.wantErr, err.Error())
+				for _, wantMissingParam := range tt.wantMissingParams {
+					assert.Contains(t, err.Error(), wantMissingParam)
```

Comment:
Welcome to the dark side
![giphy 12](https://user-images.githubusercontent.com/613317/27401377-ec3f05b8-5677-11e7-845b-13e3e6f116ed.gif)